packages/block-library/src/archives/index.php

```diff
@@ -68,7 +68,7 @@ function render_block_core_archives( $attributes ) {
 
 		$label = esc_attr( $label );
 
-		$block_content = '<label class="screen-reader-text" for="' . $dropdown_id . '">' . $title . '</label>
```

Comment:
Is this necessary? Per https://github.com/WordPress/gutenberg/issues/10871#issuecomment-434289923 it looks like we may want to keep this.

Reply:
So I think I got a little eager with this PR, but essentially I felt it was necessary to make the labels visible because it will provide better context to all users. If we can get some design feedback, I can easily revert the removal of these classes. There was some discussion about this in the original issue for this. Maybe @afercia can also speak to why others felt it made sense to make the label visible.